app/models/katello/ping.rb

```diff
@@ -14,7 +14,7 @@ def pulpcore_enabled # for downstream 6.9, remove in 6.10
 
       def services(capsule_id = nil)
         proxy = fetch_proxy(capsule_id)
-        services = [:candlepin, :candlepin_auth, :foreman_tasks, :katello_events, :candlepin_events, :katello_agent]
+        services = [:candlepin, :candlepin_auth, :foreman_tasks, :katello_events, :candlepin_events]
         services += [:pulp3] if proxy&.pulp3_enabled? && pulpcore_enabled
```

Comment:
Could we add a line like this one for katello_agent inclusion?

Reply:
yeah, will update it, i thought you guys wanted it totally removed, will add it back if the agent is present.